spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -155,6 +155,13 @@
 
 	private static final SpelExpressionParser PARSER = new SpelExpressionParser();
 
+	/*
+	 * Not static as normal since we want this TL to be scoped within the template instance.
```

Comment:
???
Please, elaborate more.
The `ThreadLocal` is stateless and really just a delegate to the `Thread.currentThread()`